### PR TITLE
[Dashboard] Remove chakra-ui from snapshot-upload

### PR DESCRIPTION
**Pull request by kien-ngo**
part of DASH-388

---

## PR-Codex overview
This PR focuses on updating the return types in the `convertFiatToCrypto` and `convertCryptoToFiat` functions to return objects instead of plain numbers. Additionally, it refactors the `SnapshotUpload` component to replace `Flex` components with `div` elements and update `Link` components to `a` elements for better semantics.

### Detailed summary
- Changed the return type of `convertFiatToCrypto` to `{ result: number }`.
- Changed the return type of `convertCryptoToFiat` to `{ result: number }`.
- Replaced `Flex` components with `div` elements in `SnapshotUpload` for layout.
- Updated `Link` components to `a` elements in `SnapshotUpload` for semantic correctness.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by kien-ngo** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5543?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#5543** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5543?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5543?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* **#5540** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5540?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/_components/claim-conditions/snapshot-upload.tsx [hunk 1]
```diff
@@ -159,13 +158,7 @@ export const SnapshotUpload: React.FC<SnapshotUploadProps> = ({
...
             />
           ) : (
-            <Flex
-              flexGrow={1}
-              align="center"
-              overflow="auto"
-              gap={8}
-              flexDir="column"
-            >
+            <div className="grow flex-col items-center gap-8 overflow-auto">
```
[the remark this answers is not in the document]
Comment (MananTank) [reply apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/_components/claim-conditions/snapshot-upload.tsx:161]: @kien-ngo Please address this - everything else LGTM

**Review by MananTank — COMMENTED** [review]
@kien-ngo please attach linear ticket reference

File: apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/_components/claim-conditions/snapshot-upload.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (kien-ngo) [reply apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/_components/claim-conditions/snapshot-upload.tsx]: updated!

File: packages/thirdweb/src/pay/convert/fiatToCrypto.ts [hunk 2]
```diff
@@ -54,7 +54,7 @@ export type ConvertFiatToCryptoParams = {
  *   fromAmount: 0.02,
  * });
  * ```
- * Result: `0.0000057` (a number)
+ * Result: `{ result: 0.0000057 }`
```
Comment (kien-ngo) [inline packages/thirdweb/src/pay/convert/fiatToCrypto.ts:57]: Need to sneak in this fix!

File: packages/thirdweb/src/pay/convert/cryptoToFiat.ts [hunk 3]
```diff
@@ -51,7 +51,7 @@ export type ConvertCryptoToFiatParams = {
  *   fromAmount: 1,
  * });
  *
- * // Result: 3404.11
+ * // Result: `{ result: 3404.11 }`
```
Comment (kien-ngo) [inline packages/thirdweb/src/pay/convert/cryptoToFiat.ts:54]: Need to sneak in this fix!

**Pull request merged**